src/components/UI/Icons/IconCheck.tsx

```diff
@@ -0,0 +1,24 @@
+import IconProps from '@/types/IconProps'
+import { ICON_SIZE } from '@/types/enums/IconSize'
+import { FC } from 'react'
+
+const IconCheck: FC<IconProps> = ({ className, size }) => {
+  const dimension = size ? ICON_SIZE[size] : ICON_SIZE['md']
```

Comment:
Seems this line will be always copy-pasted and it is bad.

Reply:
For now I'll just create a misc utility for getting the dimension